Makefile

```diff
@@ -19,40 +19,49 @@ dev: ## Setup a development environment
 
 .PHONY: build
 build: ## Build the project, including non installable libraries and executables
-	dune build --root .
+	dune build @all
```

Comment:
Is there a difference between `dune build` and `dune build @all`?

Reply:
The former builds the default target (`@all` since 2.0). The latter builds the `@all` target. I guess we should build the default target.

Note that what we had before is `$ dune build .` which means build all the targets in `.`